projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -68,16 +82,37 @@ export class NavigationListComponent implements OnChanges {
   @Output()
   public readonly collapsedChange: EventEmitter<boolean> = new EventEmitter();
 
+  public usePageLevelTimeRange: boolean = false;
+
   public activeItem$?: Observable<NavItemLinkConfig | undefined>;
 
+  public navItems$?: Observable<NavItemConfig[]>;
+
   public constructor(
     private readonly navigationService: NavigationService,
     private readonly activatedRoute: ActivatedRoute,
-    private readonly navListComponentService: NavigationListComponentService
+    private readonly navListComponentService: NavigationListComponentService,
+    private readonly timeRangeService: TimeRangeService,
+    private readonly featureStateResolver: FeatureStateResolver
   ) {}
 
   public ngOnChanges(): void {
     this.navItems = this.navListComponentService.resolveFeaturesAndUpdateVisibilityForNavItems(this.navItems);
+
+    this.navItems$ = this.featureStateResolver
+      .getCombinedFeatureState([ApplicationFeature.PageTimeRange, ApplicationFeature.NavigationRedesign])
+      .pipe(
+        map(featureState => featureState === FeatureState.Enabled),
+        tap(usePageLevelTimeRange => (this.usePageLevelTimeRange = usePageLevelTimeRange)),
```

Comment:
Avoid tap. Looks like this is used to determine whether a click handler should fire or not, but that already behaves correctly if time range isn't assigned, as determined by this same evaluation. In other words, this side effect could be removed with no further changes (other than removing the definition and read)